python/mirascope/llm/responses/base_response.py

```diff
@@ -63,5 +63,36 @@ def format(self) -> FormatT:
         raise NotImplementedError()
 
     def pretty(self) -> str:
-        """Return a string representation of all response content."""
-        raise NotImplementedError()
+        """Return a string representation of all response content.
```

Comment:
Moved the implementation from response to base_response so it can be accessible on stream_response too